WikipediaUnitTests/ArticleTestHelpers.swift

```diff
@@ -0,0 +1,450 @@
+        guard let htmlURL = URL(string: "https://en.wikipedia.org/api/rest_v1/page/mobile-html/United_States"),
+            let uniqueHTMLFileName = fileNameGenerator.uniqueFileNameForURL(htmlURL, type: .article),
+            let uniqueHTMLHeaderFileName = fileNameGenerator.uniqueHeaderFileNameForURL(htmlURL, type: .article) else {
+                XCTFail("Failure determining html file name")
+                return
+        }
+        
+        guard let cssURL = URL(string: "https://en.wikipedia.org/api/rest_v1/data/css/mobile/site"),
+            let uniqueCSSFileName = fileNameGenerator.uniqueFileNameForURL(cssURL, type: .article),
+            let uniqueCSSHeaderFileName = fileNameGenerator.uniqueHeaderFileNameForURL(cssURL, type: .article) else {
+                XCTFail("Failure determining html file name")
+                return
+        }
+        
+        guard let imageURL = URL(string: "https://upload.wikimedia.org/wikipedia/en/thumb/a/a4/Flag_of_the_United_States.svg/960px-Flag_of_the_United_States.svg.png"),
+            let uniqueImageFileName = fileNameGenerator.uniqueFileNameForURL(imageURL, type: .image),
+        let uniqueImageHeaderFileName = fileNameGenerator.uniqueHeaderFileNameForURL(imageURL, type: .image) else {
+                XCTFail("Failure determining image file name")
+                return
+        }
+        
+        guard let fixtureData = self.fixtureData else {
+                XCTFail("Failure pulling data from Fixtures")
+                return
+        }
+        
+        //save content in file system
+        CacheFileWriterHelper.saveData(data: fixtureData.cachedHTML, toNewFileWithKey: uniqueHTMLFileName) { (result) in
+            switch result {
+            case .success, .exists:
+                break
+            case .failure:
+                XCTFail("Failure saving html data")
+            }
+        }
+        
+        CacheFileWriterHelper.saveData(data: fixtureData.image, toNewFileWithKey: uniqueImageFileName) { (result) in
+            switch result {
+            case .success, .exists:
+                break
+            case .failure:
+                XCTFail("Failure saving html data")
+            }
+        }
+        
+        CacheFileWriterHelper.saveData(data: fixtureData.cachedCSS, toNewFileWithKey: uniqueCSSFileName) { (result) in
+            switch result {
+            case .success, .exists:
+                break
+            case .failure:
+                XCTFail("Failure saving html data")
+            }
+        }
+        
+        //save headers in file system
+        let htmlHeaders: [String: String] = ["Content-Type": "text/html; charset=utf-8;"]
+        
+        CacheFileWriterHelper.saveResponseHeader(headerFields: htmlHeaders, toNewFileName: uniqueHTMLHeaderFileName) { (result) in
+            switch result {
+            case .success, .exists:
+                break
+            case .failure:
+                break
```

Comment:
It looks like there is an `XTCFail` above when the call fails. Out of curiosity, is there a reason we don't want the same behavior here?

Reply:
Don't think there was a reason for this, probably a miss and a a copy-paste propagation. These should be fixed now.